content/usecases.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Use Cases"
+date: 2018-10-08T00:00:00-04:00
+draft: false
+layout: faq
+description: 
+---
+
+                                                                                                                                                                                                                                                                                                                                                                             
+-------------------------------
+Cool uses for qri: Publish, Sync, Collaborate, Inter-operate & Audit
+
+
+**Publish**
+
+_Qri draws attention to your work._
+
+Put your data directly into the hands of those who want to see it and use it.
+
+Example:
+We built a dataset from Canadian Football League transactions to show injury rates over time by team.  We built in a visualization to make the data easy to interpret, and included the script used to generate the dataset, so anyone can see how it was converted from its original publication online into qri. This data is now easy to discover, use and update for professional and amateur researchers alike. 
```

Comment:
Rather than talk about "datasets we've built" I think we should lead with a link to the dataset on app.qri.io and describe how this CFL sports stats dataset creates structured data from a raw HTML source. The idea would be to get a headline going for a sports stats enthusiast, and show how datasets could match their interests, even if they don't necessarily care about Canadian Football 😄